src/rebar_prv_common_test.erl

```diff
@@ -9,7 +9,7 @@
          do/1,
          format_error/1]).
 %% exported for test purposes, consider private
--export([compile/2, prepare_tests/1, translate_paths/2]).
+-export([compile/2, prepare_tests/1, translate_paths/2, maybe_write_coverdata/1]).
```

Comment:
if it's used only in tests, it may make sense to only export that function within an `-ifdef(TEST).` block.